app/controllers/generic_controller.rb

```diff
@@ -203,6 +203,10 @@ def base_index_ransack_query
   end
 
   def base_index_database_query
+    # Use reorder to override the default ordering in the scope.
```

Comment:
Something seems to have gone wrong here, since it does not actually use reorder. However, I've changed this code in my upcoming pull request so might as well leave it like this for now.

Reply:
You did it for the home page only I think. This is more generic.